src/routes/trip.route.js

```diff
@@ -76,4 +76,39 @@ router.post(
   TripMiddleware.checkTripExist, TripMiddleware.multiCityDataValidation,
   tripController.combineTripsConctroller
 );
+
+/**
+ * @swagger
+ *
+ * /trips/myrequests:
+ *   get:
+ *     summary: User can see all requests he/she submitted to barefoot nomad
+ *     tags: [Trips]
+ *     description: All User Requests
+ *     produces:
+ *       - application/json
+ *     parameters:
+ *       - name: token
+ *         in: header
+ *         description: Check token authentication
+ *         required: true
+ *         type: string
+ *       - name: page
+ *         in: path
+ *         description: page number
+ *         required: true
+ *         type: string
+ *     responses:
+ *       200:
+ *         description: "Successful operation"
+ *       400:
+ *         description: "Bad request"
+ *       401:
+ *         description: "Unauthorized"
+ *       409:
+ *         description: "Conflict"
+ *
+ * */
+router.get('/myrequests', verifyToken.headerToken, verifyUser, tripController.getTripRequestsByUser);
```

Comment:
Hey @Jajabenit250 wouldn't it be much better to name this route as */users/:userID/trip-requests*?.